packages/rollup-services/exec/build_and_run.sh

```diff
@@ -7,19 +7,21 @@ set -e
 
```

Comment:
This file is not part of the production codepath, it is only called from the entrypoint in `Dockerfile.dev`